modules/ROOT/pages/mule-oauth-provider-landing-page.adoc

```diff
@@ -12,13 +12,137 @@ endif::[]
+    							clientSecurityProvider="clientSecurityProvider"
+    							supportedGrantTypes="AUTHORIZATION_CODE,IMPLICIT,RESOURCE_OWNER_PASSWORD_CREDENTIALS,CLIENT_CREDENTIALS"
+    							listenerConfig="https.listener"
+    							scopes="CONTRACTOR, PUBLIC_READ, EMPLOYEES_ONLY, WRITE"
+                                defaultScopes="CONTRACTOR, PUBLIC_READ, EMPLOYEES_ONLY, WRITE"
+                                clientStore="clientObjectStore">
+    		<oauth2-provider:token-config path="/access-token" tokenTtl="5" tokenTtlTimeUnit="SECONDS" tokenStore="tokenObjectStore">
+                <oauth2-provider:refresh-token-strategy>
+                    <oauth2-provider:single-refresh-token objectStore="refreshTokenObjectStore"/>
+                </oauth2-provider:refresh-token-strategy>
+            </oauth2-provider:token-config>
+    		<oauth2-provider:authorization-config path="/authorize" authorizationCodeStore="authorizationCodeObjectStore"/>
+    	</oauth2-provider:config>
+
+        <flow name="validateToken">
+            <http:listener path="/validate" config-ref="https.listener"/>
+            <oauth2-provider:validate-token scopes="#[['PUBLIC_READ']]" config-ref="external-oauth2-provider" accessToken="#[attributes.queryParams.access_token]"/>
+            <error-handler>
+                <on-error-continue type="OAUTH2-PROVIDER:TOKEN_UNAUTHORIZED">
+                    <set-payload value="UNAUTHORIZED_TOKEN"/>
+                </on-error-continue>
+            </error-handler>
+        </flow>
+...
+----
+
+For complete information, see https://anypoint.mulesoft.com/exchange/org.mule.templates/api-gateway-oauth2-provider/[Mule OAuth 2.0 server for Anypoint Platform].
+
+== Client Application Cache
+
+To prevent any possible downtime of the Mule OAuth 2.0 provider due to errors when connecting to Anypoint Platform, the Mule OAuth Client Store caches each valid Client Application for which a token is requested.
+
+In the rare instance that the provider requests a new token when there is no connection available, the client store falls back to cache. The entries expire in 30 days, which is the default setting. To ensure service availability in case of a restart or lack of connection to Anypoint Platform, you can configure this range to persist the cache locally.
+
+In case a client application for which a token already exists in the client store is removed from Anypoint Platform,
+the client store removes the client from the cache on the next token request, mirroring the platform status. The mechanism that caches the result of a successful token validation is part of the xref:external-oauth-2.0-token-validation-policy.adoc[Mule OAuth 2.0 access token enforcement policy].
+
+=== Customizing the Client Application Cache Mechanism
+
+The customization of the default values of the cache mechanism is outlined in this section. This feature is available for Mule 4.2.0 and later.
```

Comment:
Why do we need the first sentence, which is self-referential?
Suggest replacing with something meaningful, or omitting:
```suggestion
The customization of the default values of the cache mechanism is available in Mule 4.2.0 and later.
```

Reply:
Done! Thanks!